naloge/maphija.tex

```diff
@@ -25,5 +25,59 @@
 Zakaj?
 \end{vprasanje}
 \begin{odgovor}
+
+Definiramo si spremenljivke, ki jih bomo potrebovali in izpišemo znane podatke.
+\begin{itemize}
+\item[] $X$ \dots zaslužek
+\item[] $A$ \dots recept uspe
+\item[] $B$ \dots prosijo za nasvet
+\item[] $C$ \dots svetujejo ugodno
+\item[] $D$ \dots sami proizvajajo
+\end{itemize}
+\begin{align*}
+&P(A) = \frac{2}{5} && P(\neg A) = \frac{3}{5}\\
+&P(C|A) = \frac{3}{4} & &P(C|\neg A) = \frac{1}{2}\\
+&P(\neg C|A) = \frac{1}{4} & &P(\neg C| \neg A) = \frac{1}{2}
+\end{align*}
```

Comment:
Tudi tukaj poskrbita, da so poravnani enačaji. Lahko uporabita okolje `align*` - enačbe v isti vrstici ločita z `&`, pa se bo vmes vstavil ustrezen prostor.